DRAFT.md

```diff
@@ -867,12 +869,12 @@ class indirect {
   template <class U, class AA>
   friend constexpr auto operator<=>(
     const indirect& lhs, const indirect<U, AA>& rhs) noexcept(see below)
-    -> compare_three_way_result_t<T, U>;
+    -> synth-three-way-result<T, U>;
```

Comment:
I thought we had previously made this change in the LWG session in Tokyo.  This is want we want.